hive/pubspec.yaml

```
@@ -8,8 +8,9 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  meta: ^1.3.0
   crypto: ^3.0.0
+  meta: ^1.3.0
+  js: ^0.6.4
```

```suggestion
  js: ^0.6.3
```

Can we downgrade this to `^0.6.3`. 0.6.4 requires min SDK version to be 2.14 (ours is 2.12 currently).